stablehlo/reference/Element.cpp

```diff
@@ -350,6 +350,30 @@ Element min(const Element &e1, const Element &e2) {
       });
 }
 
+Element rem(const Element &e1, const Element &e2) {
+  return map(
+      e1, e2,
+      [&](APInt lhs, APInt rhs) {
+        return isSupportedSignedIntegerType(e1.getType()) ? lhs.srem(rhs)
+                                                          : lhs.urem(rhs);
+      },
+      [](bool lhs, bool rhs) -> bool {
+        llvm::report_fatal_error("rem(bool, bool) is unsupported");
+      },
+      [](APFloat lhs, APFloat rhs) {
+        // std::fmod VS std:remainder: the returned value of the latter is not 
```

Comment:
"APFloat::mod vs APFloat::remainder"?

Reply:
I meant the latter, but seems APFloat is more clear here. Changed.